packages/x-components/src/x-modules/tagging/wiring.ts

```diff
@@ -71,5 +99,8 @@ export const taggingWiring = createWiring({
   },
   QueryTaggingDebounceProvided: {
     setQueryTaggingDebounce
+  },
+  SearchTaggingChanged: {
+    trackTaggingAction: trackTaggingWire
```

Comment:
As suggested above, just use the wire directly, there's no need to create another key for it.

```suggestion
    trackWire
```